test/payment/v1/payment-mappers/payment-mapper.spec.js

```diff
@@ -191,6 +211,57 @@ describe('PaymentMapper', () => {
         );
     });
 
+    it('maps requests for instrument to be vaulted AND made default', () => {
```

Comment:
QQ What's the difference between this and above test at line 177?
Do we want one for using existing stored instrument and one for using brand new credit card?

Reply:
There is a fair amount of overlap, I mostly included it to offer extra contrast to `'ignores requests to set non-vaulted instruments as default'`